data/duke.txt

```diff
@@ -0,0 +1,5 @@
+T|1|balls
+T|0|sell book
+D|0|return book |2/12/2019 1800
+T|0|balls
+T|0|isha
```

Comment:
Perhaps include this in the `.gitignore`, so that your stored data doesn't get included in the version control?